README.md

```diff
@@ -1 +1,156 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Hello-Microverse</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+**[Hello-Microverse]** This project is a Hello Microverse repository. No complex coding is required for this exercise. The goal is to master all of the tools and best practices learned in previous steps. They will be used in all Microverse projects and most likely in future jobs as well, so it is important to know!.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>front end</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+- added index.html 
+- added style.css
+
+- **[Display Hello-Microverse]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1- web browser
+2- git-syn
+3- code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+https://github.com/Young-Gaius/Hello-Microverse
+
+### Install
+
+Install this project with:
+
+...sh
+  cd my-project
+  npm install
+  ...
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using the the live-server
+
+### Run tests
+
+To run tests, run the following command:
+
+
+Example command:
+[ ] coming soon
+
+### Deployment
+
+You can deploy this project using:
+
+[ ] coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] To make your README to be more professional and descriptive to non-technical users, I think it would be better if you improve information from the `Getting Started` section by adding something like how to `clone` your repo, `access` the cloned directory, `install` dependencies(such for linters), `use/run` your application locally, `run tests`(your linters tests),... I recommend you follow this **[Gist template](https://gist.github.com/tresorsawasawa/9e65422376a395c17d79bcb10c31077e)** as an example to help you complete everything. If you don't have information related to your project to fill in any sub-section I suggest you completetly remove it from your docuentation and add it later in case you have something to put there instead of leaving a messgae saying `coming soon....`